code/game/atoms_movable.dm

```diff
@@ -237,7 +237,7 @@
 		pulledby.stop_pulling()
 
 	// They are moving! Wouldn't it be cool if we calculated their momentum and added it to the throw?
-	if(thrower && thrower.last_move && thrower.client && thrower.client.move_delay >= world.time + world.tick_lag * 2)
+	if(thrower && thrower.last_move && isliving(thrower) && thrower.client && thrower.client.move_delay >= world.time + world.tick_lag * 2)

```

Comment:
This is adding sanity where it's not needed, still.

Look at the *code* and how it's parsed and what its arguments are and how they're used to determine a problem; if you do that, then it makes it easier to debug things and you can avoid adding performance overhead or checks like this where they're *not necessary*.

You shouldn't be attempting to pass a mob argument to `thrower` in the first place.